docs/setting-up/expo.md

```diff
@@ -75,6 +75,13 @@ Then run the following to generate the native project directories. Run this comm
 npx expo prebuild --clean
 ```
 
+This will generate native project directories. [Expo recommends](https://docs.expo.dev/workflow/continuous-native-generation/#usage-with-eas-build) adding these directories to your `.gitignore` if you haven't already:
+```sh
+/android
+/ios
+```
+
+
```

Comment:
```suggestion
```

Thanks, this is a good advice but I want to keep the guide focused, and avoid too many details.

Expo should update their default template to include those directories in gitignore by default.